src/community/ogcapi/dggs/dggs-clickhouse/src/test/java/org/geotools/dggs/clickhouse/ClickHouseGeometrylessOnlineTest.java

```diff
@@ -28,7 +28,7 @@ protected JDBCGeometrylessTestSetup createTestSetup() {
     }
 
     protected HashMap createDataStoreFactoryParams() throws Exception {
-        HashMap params = super.createDataStoreFactoryParams();
+        HashMap params = new HashMap(super.createDataStoreFactoryParams());
```

Comment:
This part conflicts with master, cannot merge because of it.

Reply:
fixed